### DISCUSSION TITLE
Graphs gone

### DISCUSSION THREAD

**Discussion by czollars** [poblabs/weewx-belchertown#896] [Q&A]
Has anyone seen their graphs just go away after years of working correctly?  I have lost my home page graphs.  I deleted graphs.conf but no luck so far.  I copied and pasted a new graphs.conf.example but this doesn't seem to help.  Stuck at this point.

**Comment by poblabs** [comment]
Highcharts version changed. Upgrade to skin 1.3.1 which goes back to the old highcharts version https://github.com/poblabs/weewx-belchertown/releases/tag/weewx-belchertown-1.3.1

**Reply by czollars** [reply]
Thanks Pat.  Do you think this will have any affect on the web cam tab you built go next to the radar page?  Years ago, you gave me a Belchertown beta version that had a built in webcam tab and it has worked perfect ever since.  It wasn't part of the original skin so I don't want to loose it.

**Reply by poblabs** [reply]
I don't recall necessarily, but take a look at https://belchertownweather.com since that's running the latest. See if it looks like what you're saying?

**Reply by czollars** [reply]
![image](https://github.com/poblabs/weewx-belchertown/assets/137193016/487c31d3-636f-44e6-8eca-d82067e5783f)
Yes, that is it.  I remember  you saying years ago that the web cam tab next to the radar was still in beta mode.  But it looks like  it is all part of the skin now.

**Reply by czollars** [reply]
Is the safest way to do this is to move the current skin to a Belchertown-old folder and just reinstall 1.3?

**Reply by poblabs** [reply]
Yes, I would stop weewx, maybe download the whole folder so you have a point-in-time backup of everything. Then you can rename the skin folder, and also rename the belchertown.py file in the bin/user folder. Then you can install the skin as outlined with the wee_extension command. Then start weewx again

**Reply by czollars** [reply]
I'll see if I can copy the whole folder from the Raspberry Pi over to a separate SD card.  That's probably the safest way.
